sqlpage.service

```diff
@@ -1,50 +1,41 @@
-# This is a basic systemd service file for SQLPage
-# For more information about systemd service files, see https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html
-# Put this file in /etc/systemd/system/sqlpage.service
+# SQLPage systemd service file for manual/source installations
+# For package installations, see debian/sqlpage.service
+# Install to: /etc/systemd/system/sqlpage.service
 
 [Unit]
-Description=SQLPage website
+Description=SQLPage Web Server
 Documentation=https://sql-page.com
 After=network.target
 
 [Service]
-# Define the user and group to run the service
+Type=simple
 User=sqlpage
 Group=sqlpage
-
-# Set the working directory and the executable path
 WorkingDirectory=/var/www/sqlpage
 ExecStart=/usr/local/bin/sqlpage.bin
 
-# Environment variables
 Environment="RUST_LOG=info"
 Environment="LISTEN_ON=0.0.0.0:80"
```

Comment:
First it must be better to restrict to 127.0.0.1, if an user use this file on a VPS and don't have protect their page, a scan can stole datas quickly, Second we have often a webserver like nginx or apache install by default and port 80 is already used, maibe to use another one (like 8880)